apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/ConnectionPoolOptions.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Google.Cloud.Spanner.V1;
+using Google.Cloud.Spanner.V1.Logging;
+
+namespace Google.Cloud.Spanner
+{
+    /// <summary>
+    /// </summary>
+    public sealed class ConnectionPoolOptions
+    {
+        private ConnectionPoolOptions()
+        {
+        }
+
+        /// <summary>
+        ///     The default instance of connection pool options.
+        /// </summary>
+        public static ConnectionPoolOptions Instance { get; } = new ConnectionPoolOptions();
+
+        /// <summary>
+        /// </summary>
+        public TimeSpan KeepAliveTimeSpan { get; set; } = TimeSpan.FromMinutes(55);
+
+        /// <summary>
+        ///     Sets the log level for diagnostic logs sent to Trace (for desktop) or stderr (for .Net Core).
+        /// </summary>
+        public LogLevel LogLevel
+        {
+            get
+            {
+                switch (Logger.LogLevel)
```

Comment:
Any reason not to make our log levels line up with the ones in the proto, and then we can just cast? (We still need to check whether it's a known value, admittedly - we could either collapse all case labels together, or call Enum.IsDefined.)